docs/app/Examples/modules/Dropdown/Usage/index.js

```diff
@@ -120,9 +120,9 @@ const DropdownUsageExamples = () => (
       examplePath='modules/Dropdown/Usage/DropdownExampleUpwardSelection'
     />
     <ComponentExample
-      title='Wrap Selection'
-      description='A dropdown can wrap its selection to disable "infinite" selection'
-      examplePath='modules/Dropdown/Usage/DropdownExampleWrapSelection'
+      title='Unwrap Selection'
```

Comment:
Let's keep this named after the prop, "Wrap Selection".

Reply:
Ok cool. What about the description and the component name, are they fine?

Reply:
```jsx
description={[
  'This is one line of description that is quite long, however, there is even',
  ' more to go.  Luckily, we can just use an array of strings and join them!'.
].join('')}
```

Reply:
Oh, I see, no problem ;P

Will update that later today